Formula/sqlite.rb

```diff
@@ -8,7 +8,7 @@ class Sqlite < Formula
 
   livecheck do
     url "https://sqlite.org/news.html"
-    regex(%r{v?(\d+(?:\.\d+)+)</h3>}i)
+    regex(%r{v?(\d+(?:\.\d+)+)(</a>)?</h3>}i)
```

Comment:
I suppose we could also do
```ruby
regex(%r{v?(\d+(?:\.\d+)+).*</h3>}i)
```
here, but aiming for a less permissive regex is usually a good idea.

Reply:
```suggestion
    regex(%r{href=.*?releaselog/v?(\d+(?:[._]\d+)+)\.html}i)
    strategy :page_match do |page, regex|
      page.scan(regex).map { |match| match&.first&.gsub("_", ".") }
    end
```